prov/gni/test/cq.c

```diff
@@ -166,12 +168,36 @@ void cq_wait_mutex_cond_setup(void)
 	cq_create(FI_CQ_FORMAT_MSG, FI_WAIT_MUTEX_COND, 8);
 }
 
+
+void cq_notify_setup(void)
+{
+        int ret;
+
+        setup();
+
+        ret = fi_endpoint(dom, fi, &fidep, NULL);
+        cr_assert(!ret, "fi_endpoint");
+
+        ep = container_of(fidep, struct gnix_fid_ep, ep_fid);
+        cr_assert(ep, "ep not allocated");
```

Comment:
There are two reasons why this is unnecessary and/or bad practice. If fi_endpoint returned FI_SUCCESS, but returned a NULL pointer to fidep AND said structure had the fid object at the base of the structure, then it is the only case that would be caught by this statement. 

However, if the fid isn't located at the base address of the structure, then 0x0 - some_offset, would give something the cr_assert would not catch. 

For the sake of consistency within the test suites, our libfabric items are typically just the name of the item and have a '_' between fid and the item name. Please change the name, and mark fidep as a static variable so that it is not exposed outside of the scope of this file.